apps/build-onchain-apps/pages/api/README.md

```diff
@@ -0,0 +1,11 @@
+# Build Onchain API Routes
+
+## Supported Chains /api/chains/supported
+
+**Method**: GET
+
+**Response**: JSON encoded viem chain object
+
+```json
+[{"id":5,"network":"goerli","name":"Goerli",...]
```

Comment:
we should change this to base-goerli maybe? Thoughts? rest of the app currently uses that as an example

Reply:
Also maybe we should also add this [here](https://github.com/coinbase/build-onchain-apps/blob/main/README.md)?

Reply:
It is base-goerli :) see chainConfiguration.ts this is just an example payload.  I didn't want to clutter the main readme.